CRM/Report/Form.php

```diff
@@ -2558,6 +2558,7 @@ public function formatDisplay(&$rows, $pager = TRUE) {
    */
   protected function alterStateProvinceID($value, &$row, $selectedfield, $criteriaFieldName) {
     $url = CRM_Utils_System::url(CRM_Utils_System::currentPath(), "reset=1&force=1&{$criteriaFieldName}_op=in&{$criteriaFieldName}_value={$value}", $this->_absoluteUrl);
+    $row[$selectedfield . '_pristine'] = $value;
```

Comment:
SearchKit and Views use the word "raw" instead of "pristine" so maybe that's a more standard term to use?

Reply:
@colemanw sure, will make the changes to standardize the nomenclature. Thanks!

Reply:
@yashodha Are you still working on this one?